waiter/src/waiter/request_log.clj

```diff
@@ -93,7 +93,10 @@
 (defn log-request!
   "Log a request"
   [request response]
-  (log (merge (request->context request) (response->context response))))
+  (let [redacted-request-fields-string (get-in response [:descriptor :service-description "env" "WAITER_CONFIG_REDACTED_REQUEST_FIELDS"])
+        redacted-request-fields (when-not (str/blank? redacted-request-fields-string)
+                                  (map keyword (str/split redacted-request-fields-string #",")))]
+    (log (apply dissoc (merge (request->context request) (response->context response)) redacted-request-fields))))
```

Comment:
Was this already discussed: removing the keys vs replacing the values with a special string like `_REDACTED_`?

Reply:
discussed offline